doc/installation.rst

```diff
@@ -262,6 +256,7 @@ The recommended configuration is achieved by running the following command
       --enable-pmc
 
 If the ``configure`` script finds any problems with your system, it will complain loudly.
+The flag ``with-boost-python-suffix`` might be necessary, depending on the installation of Python and BOOST.
```

Comment:
At least this is the case on macOS, where you install `boost-python3` via brew.

Reply:
Specifically, check for the available library version with `ls /usr/local/lib/libboost_python*`, resulting in e.g. `/usr/local/lib/libboost_python39.a`. Then use `--with-boost-python-suffix=39`.

Reply:
I agree with the second comment.